clcache/__main__.py

```diff
@@ -1739,7 +1763,8 @@ def scheduleJobs(cache: Any, compiler: str, cmdLine: List[str], environment: Any
     cleanupRequired = False
 
     def poolExecutor(*args, **kwargs) -> concurrent.futures.Executor:
-        if isTrackerEnabled():
+        toolsetVersion = findToolsetVersion(findCompilerVersion(compiler))
+        if isTrackerEnabled() and toolsetVersion < 140:
```

Comment:
As mentioned by @frerich at https://github.com/izmmisha/clcache/pull/5#issuecomment-521637705, would be better to check compiler version only when `isTrackerEnabled()` is `True`.
Right now it will check version of compiler if tracker is not detected too.